### ISSUE TITLE
get_delta_stats() bug 

### ISSUE THREAD

**Issue by dmcglinn** [MoBiodiv/mobr#33]
https://github.com/MoBiodiv/mobr/blob/master/R/mobr.R#L466 is causing an error

```
Error in colSums(x) : 'x' must be an array of at least two dimensions
```

not sure why yet

**Comment by dmcglinn** [comment]
note that the mite example works great this is for one of the forest anaylses

**Comment by dmcglinn** [comment]
this error occurs because the object `comm$comm` does not have `row.names` they are simply set to `NULL`. I'll consult with @rueuntal about patching this.

**Comment by rueuntal** [comment]
Thanks Dan! Will look into it.

**Comment by dmcglinn** [comment]
solved by #34
